### PR TITLE
SD-1607 stacktraces w/errors

### PR DESCRIPTION
**Pull request by MarkRoddy**

### CODE REVIEW

File: broker.go [hunk 1]
```diff
@@ -143,7 +145,7 @@ func (b *Broker) Close() (err error) {
 	}()
 
 	if b.conn == nil {
-		return NotConnected
+		return errors.New(NotConnected)
```
Comment (actgardner) [inline broker.go:141]: Is there a reason why only some errors are wrapped?

Comment (MarkRoddy) [inline broker.go:141]: Timeliness.  We covered the ones necessary to determine the root cause of the Fidelity issue.

**Comment by actgardner** [comment]
Looks good for the backport, for consistency we should probably wrap everything in master.

**Comment by MarkRoddy** [comment]
I started to add the additional errors, but it raises a problem.  The lib uses errors for flow control in places, which will break since we'd be returning wrapped errors.  The flow control here is already kinda nuts so I don't think it's worth teasing apart for the next release.

**Pull request merged**